java/src/main/java/com/google/appengine/tools/pipeline/di/MultiTenantComponent.java

```diff
@@ -5,9 +5,12 @@
 import javax.inject.Singleton;
 
 @Singleton // expect exactly one of these per process
-@Component
+@Component(
+  modules = {
+    AppEngineHostModule.class,
+  }
```

Comment:
previously, this was included in `TenantModule`; moving it up to `MultiTenantComponent` is what let compile work for both this and `JobRunServiceComponent`, when scope-annotations added on the `@Provides`/`@Binds` methods